README.md

```diff
@@ -62,19 +73,65 @@ You will likely need to remove "bad" characters that break bash environment vari
 
 8. Push it to CloudFoundry.
 
+Run:
+
   ```bash
   cf push
   ```
 
-9. Configure S3 WordPress Plugin
- * Login to https://mysite.app.cloud.gov/wp-admin
- * Click on AWS in the left navigation menu
- * Click on S3 and CloudFront in navigation menu
- * Turn on `Copy Files to S3` and `Rewrite File URLs`
- * Scroll to the bottom, and click Save Changes
+This will push the files in this repository to the platform (around 25k), including the buildpack extension we've adapted from the Cloud Foundry project. The server downloads and runs the [PHP Build Pack] which installs HTTPD and PHP. The buildpack sees the extension which installs [wpcli](http://wp-cli.org/), reads your `setup.json`, and installs and configures WordPress along with any plugins or themes you have defined. It then copies the `wp-config.php` file and replaces the default WordPress files with it. Once all that is finished, the platform starts the droplet and you have a WordPress site.
+
+You should see output like this in your terminal:
+
+  ```
+  App started
+
+
+  OK
+
+  App mywordpress was started using this command `$HOME/.bp/bin/start`
+
+  Showing health and status for app mywordpress in org sandbox-gsa / space gregory.boone as gregory.boone@gsa.gov...
+  OK
+
+  requested state: started
+  instances: 1/1
+  usage: 128M x 1 instances
+  urls: my-special-wordpress.app.cloud.gov
+  last uploaded: Tue Sep 26 22:21:49 UTC 2017
+  stack: cflinuxfs2
+  buildpack: https://github.com/cloudfoundry/php-buildpack
+  ```
+
+If you go to the URL listed under `urls` you should see a fresh WordPress site.
+
+
+9. Verify S3 connection
+
+This demo uses the [Human Made S3 Uploads plugin](https://github.com/humanmade/S3-Uploads), which automatically uploads files from your WordPress install to S3 and rewrites the URLs for you. The app has no configuration, the access keys, secret key, and bucket name are stored in the environment configuration and read by the plugin on start.
```

Comment:
* s/has no/requires no/
* Split the last sentence on the comma after "configuration".

Reply:
✅